### DISCUSSION TITLE
Run multiple tasks on keymap binding

### DISCUSSION THREAD

**Discussion by lio-movosoft** [zed-industries/zed#16141] [Feature Requests]
I believe it would be great if multiple commands could be run on key bindings:

```
  {
    "context": "Editor",
    "bindings": {
        "alt-h": [
          ["task::Spawn", { "task_name": "mytask that copies something to the system clipboard" }], 
          "editor:paste"
       ]
    }
  }
```
this would allow running small shell scripts to change specific text blocks like `#FF8000` to `rgb(1.0, 0.5, 0.0)` 
the task could run something like `hex2rgb $ZED_SELECTED_TEXT`

**Comment by failable** [comment]
I wish I could close the tab and delete the file in the same time...

```
"ctrl-c m k": [
        "workspace::SendKeystrokes",
        "ctrl-x ctrl-m close active item enter ctrl-x ctrl-m task spawn enter delete file enter"
]
```

**Comment by itspers** [comment]
I thought that would be a workaround to simply run two tasks, but turns out even this is not possible 😱 Now tasks in Vscode looks like 'advanced' technology, and i was schoked when it migrated from Jetbrains..
